src/metamodels/crosscat.py

```diff
@@ -240,34 +239,8 @@ class CrosscatMetamodel(metamodel.IBayesDBMetamodel):
 
     """
 
-    def __init__(self, crosscat, subsample=None, ccargs=None,
-                 cckwargs=None):
```

Comment:
I think cckwargs was so we could pass e.g. the subsample keyword to the underlying metamodel, even if we wanted to do multiprocessing. Can you help me understand how that works in the revised flow?